### PR TITLE
Game model

### PR DESCRIPTION
**Pull request by akmohtashami**

### CODE REVIEW

File: game/models.py [hunk 1]
```diff
@@ -1,65 +1,48 @@
...
-    name = models.CharField(verbose_name=_('title'), max_length=200)
-    competition = models.ForeignKey(to='game.Competition', verbose_name=_('Competition'), null=False, blank=False)
-    compile_container = models.ForeignKey('game.DockerContainer', verbose_name=_('Compile Container'), related_name='+',
-                                          null=True, blank=True)
-    execute_container = models.ForeignKey('game.DockerContainer', verbose_name=_('Execute Container'), related_name='+',
-                                          null=True, blank=True)
 
-    def __unicode__(self):
-        return self.name
+class OperationResource(models.Model):
```
Comment (akmohtashami) [inline game/models.py]: No. To define an operation some additional files might be required. These are kinda like operation static files. One might reference them in the Docker-Compose template file.

File: game/models.py [hunk 2]
```diff
@@ -1,65 +1,48 @@
...
+    operation = models.ForeignKey(Operation, verbose_name=_("operation"))
+    name = models.CharField(max_length=100, verbose_name=_("name"))
+    file = models.FileField(verbose_name=_("file"))
```
Comment (akmohtashami) [inline game/models.py]: I think it's fixed now?

File: game/models.py [hunk 3]
```diff
@@ -1,65 +1,48 @@
...
+    PARAMETER_TYPES = (
+        ('string', 'string'),
+        ('file', 'file'),
+    )
+
+    operation = models.ForeignKey(Operation, verbose_name=_("operation"))
+    name = models.CharField(max_length=100, verbose_name=_("name"))
+    type = models.CharField(choices=PARAMETER_TYPES, verbose_name=_("type"), max_length=20)
+    required = models.BooleanField(verbose_name=_("required"), default=True)
+    input_parameter = models.BooleanField(verbose_name=_("input parameter"))
```
Comment (akmohtashami) [inline game/models.py]: Done.

File: game/models.py (hunk 1 above)
Comment (altostratous) [inline game/models.py]: ok

**Pull request merged**